conn.go

```diff
@@ -62,6 +62,8 @@ type Conn struct {
 	// renegotiation extension. (This is meaningless as a server because
 	// renegotiation is not supported in that case.)
 	secureRenegotiation bool
+	//useEMS is true if we will use/have used the extended MasterSecret extension
```

Comment:
Actually, why is this used for "will"? Shouldn't that setting be part of the Config structure?